### ISSUE TITLE
[Bug] Executing PUT on unexisting resource (article) ID can trow unique title validation error

### ISSUE THREAD

**Issue by jaktestowac** [jaktestowac/gad-gui-api-demo#54]
**Describe the bug**
When triggering PUT on non-existing resource and title of article is duplication of another resource then validation reject PUT

**To Reproduce**
1. Go to Swagger, login and go to PUT articles
2. Execute PUT with non-existing id like:
```
curl -X 'PUT' \
  'http://localhost:3000/api/articles/19999' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Ik1vc2VzLkFybXN0cm9uZ0BGZWVzdC5jYSIsImRhdGEiOiJUQkQiLCJpYXQiOjE3MzM3MzM2NzQsImV4cCI6MTczMzczNzI3NH0.qdiKENenp_OqRJvzKvWDXMMhdEsVOdatq_GXib9rL0c' \
  -H 'Content-Type: application/json' \
  -d '{
  "user_id": 1,
  "title": "string1",
  "body": "string",
  "date": "2024-12-09T08:39:50.869Z",
  "image": "string"
}'
```
3. App should allow create this one with ID not matching inserted but creating new ID
4. Try to execute it once more
```
422 | Error: Unprocessable EntityResponse bodyDownload{   "error": {     "message": "Field \"title\" is not unique!"   } }

```

**Expected behavior**
New article is created with incremented ID

**Additional context**
🦎 GAD (v2.7.9)

**Comment by Freudenberger** [comment]
Fix will be introduced in upcoming version `2.7.10`

**Comment by Freudenberger** [comment]
Fix introduced in `v2.7.10`
